frontend/packages/console-dynamic-plugin-sdk/src/shared-modules.ts

```diff
@@ -32,5 +34,7 @@ export const overrideSharedModules = (entryModule: RemoteEntryModule) => {
     'react-i18next': async () => () => require('react-i18next'),
     'react-router': async () => () => require('react-router'),
     'react-router-dom': async () => () => require('react-router-dom'),
+    '@patternfly/react-core': async () => () => require('@patternfly/react-core'),
```

Comment:
@cyril-ui-developer Note that #10130 moved the `overrideSharedModules` function into a separate module (`shared-modules-override.ts`).